src/test/rgw/bucket_notification/api.py

```diff
@@ -247,17 +247,15 @@ def delete_all_topics(conn, tenant, cluster):
     if tenant == '':
         topics_result = admin(['topic', 'list'], cluster)
         topics_json = json.loads(topics_result[0])
-        if 'topics' not in topics_json:
-            topics_json = topics_json.get('result',{})
-        for topic in topics_json['topics']:
-            rm_result = admin(['topic', 'rm', '--topic', topic['name']], cluster)
-            print(rm_result)
+        if 'topics' in topics_json:
```

Comment:
you dont this 'if` statement anymore.
it was used to cober the case where you have/dont have the `topics` inside the `results` section